src/main/java/core/basesyntax/impl/CreateReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.CreateReport;
+import java.util.Map;
+
+public class CreateReportImpl implements CreateReport {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String createReport(Map<String, Integer> map) {

```

Comment:
add DB as class field
```suggestion
    public String createReport() {
```